Changes

```diff
@@ -1,5 +1,9 @@
 Revision history for Unix-PID-Tiny
 
+0.94 Wed Apr 24 16:56:09 2019
+    - declare Test::Warn, File::Slurp, File::Temp dependencies
```

Comment:
I don’t see anything about Test::Warn, File::Slurp, File::Temp in e37c8ab6296e558acbf336a2d4e7b71649ad0026 did something not get pushed?

Reply:
We were getting RPM build errors related to missing Test::Warn, File::Slurp and File::Temp.

Reply:
yeah, that I gathered, I just didn't see any change relating to those modules in e37c8ab whcih made me think something is missing. I see them now in ce81f0c 👍